cmake/OpenLocoUtility.cmake

```diff
@@ -58,14 +58,8 @@ function(loco_target_compile_link_flags TARGET)
         /W4                      # Warning level 4
                                  # Poke holes in W4 due to our interop code
         /wd4068                  #   4068: unknown pragma
-        /wd4091                  #   4091: 'keyword' : ignored on left of 'type' when no variable is declared
-        /wd4100                  #   4100: 'identifier' : unreferenced formal parameter
```

Comment:
isn't -Wno-unused-parameter the same as this. Could you remove it for gcc/clang. L72.

Reply:
I think so, I'll give it a try.